go-controller/pkg/node/node_ip_handler_linux.go

```diff
@@ -139,6 +139,22 @@ func (c *addressManager) ListAddresses() []net.IP {
 	return out
 }
 
+func (c *addressManager) ListNetworkAddresses() []*net.IPNet {
```

Comment:
does this include the IP of a localnet pod? we store those in addressManager as host IPs??

Reply:
uhm no, it's orthogonal to what we do for localnet.